packages/aws-cdk/test/api/hotswap/hotswap-test-setup.ts

```diff
@@ -16,7 +16,9 @@ export const STACK_ID = 'stackId';
 
 let hotswapMockSdkProvider: HotswapMockSdkProvider;
 let currentCfnStack: FakeCloudformationStack;
-const currentCfnStackResources: CloudFormation.StackResourceSummary[] = [];
+const currentCfnStackResources: AWS.CloudFormation.StackResourceSummary[] = [];
+let stackTemplates: {[key:string]: any};
+let currentNestedCfnStackResources: { [key: string]: AWS.CloudFormation.StackResourceSummary[] };
```

Comment:
`key` is too generic of a name. Can we give this a name that actually reflects what the keys here represent?